lib/rules/time-min-milliseconds/index.js

```diff
@@ -62,47 +62,44 @@ const rule = (primary, secondaryOptions) => {
 			}
 
 			if (shorthandTimeProperties.has(propertyName)) {
-				const valueListList = postcss.list.comma(propertyValue);
-
-				for (const valueListString of valueListList) {
-					const valueList = postcss.list.space(valueListString);
-
-					if (ignoreDelay) {
-						// Check only duration time values
-						const duration = getDuration(valueList);
-
-						if (duration && !isAcceptableTime(duration)) {
-							complain(decl, propertyValue.indexOf(duration), duration.length);
-						}
-					} else {
-						// Check all time values
-						for (const value of valueList) {
-							if (!isAcceptableTime(value)) {
-								complain(decl, propertyValue.indexOf(value), value.length);
-							}
-						}
-					}
-				}
+				const parsedValue = valueParser(getDeclarationValue(decl));
+				let isValueTimeCount = 0;
+
+				parsedValue.walk(({ value, sourceIndex }) => {
+					isValueTimeCount = valueTimeCheck(value, isValueTimeCount);
```

Comment:
[suggest] To make the purpose of the `valueTimeCheck` function clearer, how about renaming it to `calcTimeValueCount`?

```suggestion
					timeValueCount = calcTimeValueCount(value, timeValueCount);
```